documentation/src/main/asciidoc/userguide/chapters/query/spatial/Spatial.adoc

```diff
@@ -250,19 +197,12 @@ create transform for db2gse.st_geometry db2_program (
  to   sql with function db2gse.geomfromewkt(varchar(32000)) );
 ----
 
-====
-
-
-
 [[spatial-types]]
 === Types
 
 Hibernate Spatial comes with the following types:
 
-jts_geometry::
-	Handled by `org.hibernate.spatial.JTSGeometryType`, it maps a database geometry column type to a `org.locationtech.jts.geom.Geometry` entity property type.
-geolatte_geometry::
-	Handled by `org.hibernate.spatial.GeolatteGeometryType`, it maps a database geometry column type to an `org.geolatte.geom.Geometry` entity property type.
+TODO
```

Comment:
I guess you will retain the same names for something like `SpatialTypes` analogously to `StandardBasicTypes`?

Reply:
I'm not yet sure how to do it, but this whole section needs to be rewritten.